_pages/policies/employee-resources-policies/hiring-authorities.md

```diff
@@ -0,0 +1,51 @@
+  * If you have served in the **Peace Corps** for two years you are eligible to apply non-competitively. This means you are only competing against other peace corp applicants, not the general public and veterans’ preference does not apply. 
+
+## Excepted Service 
+### Schedule A(r)
+The Schedule A(r) hiring authority are term appointments for a period between one and four years.
+Employees who are appointed under term appointments do not earn competitive status. These roles can be posted on our website instead of USA Jobs.
+* There are two ways we hire using Schedule A(r): via posting a job on [JoinTTS](https://join.tts.gsa.gov/) or via [Reserved Hire](https://drive.google.com/file/d/0B4J4Dpr2HVDsX2RKMjhtU1NuU2ZEUWoxU0NJWkhLbFVXejlv/view?usp=sharing)
+  * Once a role enters the Hiring Pipeline prioritization list, Talent and the Hiring Manager discuss which avenue they would like to pursue for a given action.
+
+### Schedule A(u)
+The Schedule A(u) authority can be used to hire term or permanent employees. This hiring mechanism is used for hiring people with severe physical disabilities, psychiatric disabilities, and intellectual disabilities. Such individuals may qualify for conversion to permanent status after two years of satisfactory service. Severe physical disabilities include but are not limited to blindness, deafness, paralysis, missing limbs, epilepsy, and more.
+* There must be a hiring need identified by the head of your office to use this authority and they will determine if the role should be term or career.
+* Read more [here](https://docs.google.com/document/d/1WnvPtg6QxcrX8UQZWBZhGiRfxWphl7Nd7oy5gPXrVaQ/edit?usp=sharing)
+
+## Direct Hiring Authority (DHA)
+The DHA can be advertised as term or permanent. The Office of Personnel Management (OPM) can grant this type of hiring authority to federal agencies for filling vacancies in specific occupations, grade levels, and locations when it can be proven that there is a critical hiring need, or a severe shortage of candidates. These roles must be posted on USA Jobs. 
+* Currently, GSA has the authority to hire IT Specialists in the 2210 series via DHA until September 20, 2021. This allows us to advertise 2210 roles for an 8-year term only (no career option) and the IT Specialist (INFOSEC) role (no expiration date yet). 
+* Veteran’s preference does not apply
+* If the role was posted within the last 120 days, resumes can continue to be submitted for candidates to be found qualified against the announcement. The Position Description they are being qualified against would have to be similar to one that was announced.
+
+## Veterans’ Preference
+Veterans’ preference is a Federal hiring law that states certain veterans receive special consideration in the hiring process. It applies to nearly all competitive and many excepted service positions and is distinct
```

Comment:
The references to in this paragraph should all be Veterans' Preference, too, right?